qirlib/src/generation/emit.rs

```diff
@@ -58,7 +63,7 @@ fn build_entry_function(generator: &CodeGenerator, model: &SemanticModel) -> Res
     add_external_functions(generator, model.external_functions.iter());
     let entry_point = qir::create_entry_point(generator.context, &generator.module);
 
-    if model.static_alloc {
+    if model.use_static_qubit_alloc {
```

Comment:
I can see that this is where the `requiredQubits` attribute is written, but what about the `requiredResults` attribute, is that handled somewhere?

Reply:
@swernli didn't know that was needed, I'll add it to the spec issue and get it added

Reply:
In case it's helpful, here is the part in QAT that adds it: https://github.com/qir-alliance/qat/blob/685904d5f471bf09b3e5fe5c264adf79ad4bcf09/qir/qat/TransformationRulesPass/TransformationRulesPass.cpp#L741-L746. It uses the same syntax as `requiredQubits`.